### PR TITLE
Add an option to run Content build with latest server

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
- [ ] Ready
- [x] In code review
- [X] In Progress
- [ ] Hold
      Reason for hold:

## Description
Make Content build work with latest server Master

### CODE REVIEW

**Comment by ronykoz** [comment]
DO NOT MERGE

**Comment by yuvalbenshalom** [comment]
@ronykoz looks very good.
questions: 
1. where is the triggering script we should use to run the script? am i missing something?
2. why the mix with the integration files on the same branch?

got conflicts...

**Comment by ronykoz** [comment]
@yuvalbenshalom  
1. My bad, added the script.
2. We have couple of problems with the validator script so it did not allow me to commit these files till i do these changes, added a possible fix but need to think this through

File: Utils/trigger_content_non_ami_build.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+_circle_token=$2
+
+trigger_build_url=https://circleci.com/api/v1/project/demisto/content/tree/${_branch}?circle-token=${_circle_token}
+
+if [ -z "$3"]
+  then
+    post_data=$(cat <<EOF
+    {
+      "build_parameters": {
+        "AMI_RUN": "true"
```
Comment (yuvalbenshalom) [inline Utils/trigger_content_non_ami_build.sh]: should AMI_RUN be true in both cases?

Comment (ronykoz) [reply Utils/trigger_content_non_ami_build.sh]: Actually it should be false, but on both cases

**Comment by yuvalbenshalom** [comment]
resolve conflicts with integrations

**Pull request merged**